xbmc/utils/log.h

```diff
@@ -155,9 +155,9 @@ class CLog : public ISettingsHandler, public ISettingCallback
                                                   const char* format,
                                                   Args&&... args)
   {
-    GetInstance().FormatAndLogInternal(level,
-                                       StringUtils::Format("{0:s}: {1:s}", functionName, format),
-                                       std::forward<Args>(args)...);
+    GetInstance().FormatAndLogInternal(
+        level, StringUtils::Format("{0:s}: {1:s}", functionName, format).c_str(),
```

Comment:
Looking at this I'm wondering why `FormatAndLogInternal()` takes a `const char*` / `const wchar*` instead of a `const std::string&` / `const std::wstring&` because `StringUtils::Format()` does. I'll take a look at that ASAP.

Reply:
OK great, so I don't have to look into it :+1:

Reply:
I'll look for your opinion when I PR the changes 😄